File: lista de treino/Exercicio_3.cpp
#include<stdio.h>


int main(){

    int num     = 0;
    int cont    = 0;
    float soma  = 0;
    int ant     = 0;

    printf("digite uma sequencia de numeros em que so vai parar quando uma sequencia de 3 numeros crescentes em ordem: ");
    scanf("%d",&num);

    soma = num;

    while(cont<2){
        ant=num;
        scanf("%d",&num);
        
        if(num>ant){
            cont++;
            soma=soma+num;
        }
        
        else{
            cont=0;
            soma=num;
        }

    }

printf("A media da sequencia informada e: ");
printf("%f",soma/3);


return 0;

}

File: lista de treino/Exercicio_3.py
print('digite uma sequencia de números em que so vai parar quando uma sequência de 3 números crescentes em ordem')

num=int(input('insira um número: '))
cont=0
soma=0
ant=0
soma=num

while cont<2:
    ant=num
    num=int(input('insira um número: '))
    
    if num>ant:
        cont+=1
        soma+=num
        

    else:
        cont=0
        soma=num



print(f"A média da sequencia informada é: {soma/3}")